veto/vetoHit.cxx

```diff
@@ -1,106 +1,110 @@
 #include "vetoHit.h"
-#include "veto.h"
-#include "TVector3.h"
+
 #include "FairRun.h"
 #include "FairRunSim.h"
+#include "TGeoArb8.h"
+#include "TGeoManager.h"
 #include "TMath.h"
 #include "TRandom1.h"
 #include "TRandom3.h"
-#include "TGeoManager.h"
-#include "TGeoArb8.h"
+#include "TVector3.h"
+#include "veto.h"
 
 #include <iostream>
 #include <math.h>
 using std::cout;
 using std::endl;
 
-Double_t speedOfLight = TMath::C() *100./1000000000.0 ; // from m/sec to cm/ns
+Double_t speedOfLight = TMath::C() * 100. / 1000000000.0;   // from m/sec to cm/ns
```

Comment:
Shouldn't we take this value from shipunit?

Reply:
will remove this, I think its remnant from oldcode after rebasing with origin post clean up.